src/View/Helper/HtmlHelper.php

```diff
@@ -232,7 +232,11 @@ public function meta($type, $content = null, array $options = [])
             ];
 
             if ($type === 'icon' && $content === null) {
-                $types['icon']['link'] = 'favicon.ico';
```

Comment:
I checked and the double webroot is because assetUrl() is already used further in the code. And guess what, using this with theme already works #5687.